lib/DAV/Xml/Property/LockDiscovery.php

```diff
@@ -89,7 +89,7 @@ public function xmlSerialize(Writer $writer)
                 $writer->endElement(); // {DAV:}lockroot
             }
             $writer->writeElement('{DAV:}depth', (DAV\Server::DEPTH_INFINITY == $lock->depth ? 'infinity' : $lock->depth));
-            $writer->writeElement('{DAV:}timeout', 'Second-'.$lock->timeout);
+            $writer->writeElement('{DAV:}timeout', (LockInfo::TIMEOUT_INFINITE === $lock->timeout ? 'Infinite' : 'Second-'.$lock->timeout));
```

Comment:
```suggestion
            $writer->writeElement('{DAV:}timeout', (LockInfo::TIMEOUT_INFINITE === $lock->timeout ? 'infinite' : 'Second-'.$lock->timeout));
```

Like for depth above?